src/lib-dcrypt/dcrypt-openssl1.c

```diff
@@ -106,10 +106,20 @@
 
 #if defined(NID_X25519)
 #  define HAVE_X25519
-#  define IS_XD_CURVE(nid) \
+#  ifdef NID_X448
+#    define IS_XD_CURVE(nid)						\
 	((nid) == NID_X25519 || (nid) == NID_X448)
-# define IS_ED_CURVE(nid) \
+#  else
+#    define IS_XD_CURVE(nid)						\
+	((nid) == NID_X25519)
+#  endif
+#  ifdef NID_ED448
+#    define IS_ED_CURVE(nid)							\
```

Comment:
\ is in a bit random place. Not soon after define, not aligned with other \ either

Reply:
this is also fixed because in OpenBSD there is NID_Ed448.